Gemfile

```diff
@@ -48,7 +48,8 @@ gem 'mini_magick'
 gem 'mysql2'
 
 # Tables
-gem 'wice_grid'
+gem 'wice_grid', '3.6.0.pre4'
```

Comment:
We should keep an eye on this gem and when 3.6.0 is released, we should update the Gemfile.